Zend/tests/repeated_include/anonymous_class.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+Repeated include must not increase memory - anonymous class
+--SKIPIF--
+<?php
+// TODO test should pass even without opcache, but fixes are needed
+// related bug tracker https://bugs.php.net/bug.php?id=76982
+if (!extension_loaded('Zend OPcache') || !(opcache_get_status() ?: [])['opcache_enabled']) {
```

Comment:
This skip-if is broken. `!(opcache_get_status() ?: [])['opcache_enabled']` accesses an offset on `[]`.

Reply:
fixed

Reply:
fixed